src/connectors/github_organization.py

```diff
@@ -0,0 +1,251 @@
+"""GitHub Webhooks Connector from S3
+Already collected Webhooks in S3 being placed in Snowflake
```

Comment:
Looking at the existing Connectors, it would be better to stay consistent with "Collect {what} from {where} using {credential}" —

![image](https://user-images.githubusercontent.com/42752788/63471139-3db9d880-c423-11e9-86a2-8bf29372cc66.png)

So in this case, maybe something like —

![image](https://user-images.githubusercontent.com/42752788/63471386-dc463980-c423-11e9-9090-d4cb67b50470.png)

?

Reply:
Ok! Sounds good thanks